.github/workflows/buildpack-deps.yml

```
@@ -0,0 +1,42 @@
+name: buildpack-deps
+
+on:
+  pull_request:
+    paths:
+      - 'scripts/docker/buildpack-deps/Dockerfile.emscripten'
```

```suggestion
  pull_request:
    branches: [ develop ]
    paths:
      - 'scripts/docker/buildpack-deps/Dockerfile.emscripten'
```

Following https://github.com/ethereum/solidity/pull/9368#discussion_r453963786

Or doesn't that work for some reason?

---

For testing that you might actually merge the whole thing to ``develop`` in your fork, but that should be doable, shouldn't it :-)?